src/extractor/scripting_environment_lua.cpp

```diff
@@ -939,22 +939,37 @@ Sol2ScriptingEnvironment::GetStringListFromFunction(const std::string &function_
     return strings;
 }
 
+namespace
+{
+
+// string list can be defined either as a Set(see profiles/lua/set.lua) or as a Sequence (see
```

Comment:
It turned out we had a bug with `Set` values...

Reply:
Nice catch!
I guess that means nobody has noticed the suffixes appearing in walking guidance (or nobody is using it).